src/VisualStudio/Setup.Next/project.json

```diff
@@ -1,5 +1,9 @@
 ﻿{
     "dependencies": {
+        "Microsoft.VisualStudio.Shell.Framework": "15.0.25726-Preview5",
```

Comment:
I don't see what in this change requires these new references?  Is it because the old VSSDK added them from the product or something?